Phonebook.Frontend/src/app/pages/settings/settings.component.html

```diff
@@ -41,17 +42,15 @@ <h2 i18n="SettingsComponent|Color Theme SubTitle@@SettingsComponentSubTitleColor
           </mat-form-field>
         </mat-list-item>
       </mat-list>
-      <h2 i18n="SettingsComponent|Layout Subtitle@@SettingsComponentSubTitleLayout">Layout</h2>
+      <mat-card-subtitle class="subtitleSetting" i18n="SettingsComponent|Subtitle Dashboard of the settings page@@SettingsComponentSubtitleDashboard">
+        Dashboard</mat-card-subtitle>
```

Comment:
Can you change the title importance? So that "Dashboard" is `<h2>` and the subsettings are `mat-card-subtitle>`?